custom_components/openevse/entity.py

```
@@ -2,13 +2,31 @@
+from homeassistant.helpers.device_registry import DeviceInfo
+
+
+class OpenEVSEEntity:
+    """Base class for OpenEVSE entities."""
+
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return a port description for device registry."""
+        return DeviceInfo(
+            manufacturer="OpenEVSE",
+            name=self._config.data[CONF_NAME],
```

Pylint (4.0.6)

[error] Instance of 'OpenEVSEEntity' has no '_config' member

(E1101)